#!/bin/sh /usr/share/dpatch/dpatch-run
## 204-magic-update-cracklib.dpatch by Michael Piefel <piefel@debian.org>
##
## DP: Improved over-confident 64bit-cracklib rule (closes: #343504).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/cracklib file-4.20/magic/Magdir/cracklib
--- file-4.20.orig/magic/Magdir/cracklib	2006-03-02 22:10:26.000000000 +0000
+++ file-4.20/magic/Magdir/cracklib	2007-03-27 12:23:14.000000000 +0000
@@ -9,5 +9,5 @@
 0	belong	0x70775631	Cracklib password index, big endian
 >4	belong	>-1		(%i words)
 # really bellong 0x0000000070775631
-4	belong	0x70775631	Cracklib password index, big endian ("64-bit")
+0	search/1	\0\0\0\0pwV1	Cracklib password index, big endian ("64-bit")
 >12	belong	>0		(%i words)
